lib/mpl_toolkits/axes_grid1/axes_rgb.py

```diff
@@ -106,9 +106,9 @@ def __init__(self, *args, pad=0, add_all=True, **kwargs):
             defaults to True.
         axes_class : matplotlib.axes.Axes
 
-        kl :
+        kl
```

Comment:
what's kl here?

Reply:
That `*args` now. Has been renamed in the signature recently without updating the docstring (a68784fdd5f97d90e80c86a17551b19c0a466dbc).